f/assignGear/f_assignInsignia.sqf

```diff
@@ -32,7 +32,7 @@ switch (_insignia_style_CSAT) do
 	};
 
 // Treat CTRG like BLUFOR
-if (_faction == "blu_ctrg_f") then {_faction = "blu_f"};
+if (_faction in ["blu_ctrg_f","ctrg"]) then {_faction = "blu_f"};
```

Comment:
Should we remove this now as well and just add "blu_ctrg_f","ctrg" to the relevant cases?
Does this appear in any other files as well?

Reply:
I'd like to just stick it in the NATO case, I think that's a relic of when CTRG was the only special faction. It doesn't matter too much, though.

Reply:
Think I'd prefer CTRG to have their own cases, mainly for parity with Spetsnaz

Reply:
I'm giving CTRG their own Medic badge case, but I'm not sure what to do for their ordinary insignia. In the template, CTRG are only used as assignGear-overriden FIA units - I don't know what group IDs to assume. Assume "grpCTRG"? How often do we use that? Assume "grpNATO"? Maybe? Would appreciate input.

Reply:
Yeah, having had a look at it CTRG is a bit of a mess (treated as NATO in insignias, separately in briefing, and FIA in group data). The purist in my wants to split them out and handle them separately in all cases, and handle special cases like the template leaders by setting the Virtual Faction to "fia" and the assignGear override faction to "ctrg". This is what's going to have to be done for NPR/Spetsnaz and I think FIA/CTRG should be changed to be consistent. This would harm backwards compatibility but since no one seems to be updating missions anyway that shouldn't be a big issue.

So to more directly answer your question, I'd say CTRG should have their own case for the Medic insignia (they don't currently) and "GrpCTRG". This change doesn't need to be done here, could create a separate issue for it if you want to get this one wrapped up.